pkg/server/server_test.go

```diff
@@ -359,6 +359,46 @@ func TestLokiFiltering(t *testing.T) {
 	}, {
 		inputPath:   "?timeRange=86400000&match=any",
 		outputQuery: []string{`?query={app="netobserv-flowcollector"}&start=${timeNow-86400000}`},
+	}, {
+		inputPath: "?SrcNamespace=\"exact-namespace\"",
+		outputQuery: []string{
+			`?query={SrcNamespace=~"exact-namespace",app="netobserv-flowcollector"}`,
```

Comment:
Question, if we use the `=~` operator for `SrcNamespace`, wouldn't this query return a "contains" match?

Shouldn't be `SrcNamespace="exact-namespace"` or, at least `SrcNamespace=~"^exact-namespace$"`?

Reply:
That's right ! I will put start / end match for simplicity since value can contains `*`
 :+1: thanks